_includes/head.html

```diff
@@ -1,5 +1,6 @@
 <meta charset="utf-8"/>
 <script defer src="https://code.getmdl.io/1.3.0/material.min.js"></script>
+<script src="https://unpkg.com/lunr/lunr.js"></script>
```

Comment:
I believe you imported lunr twice. I placed another comment on the other occurrence.